core/test/gather/gatherers/dobetterweb/optimized-images-test.js

```diff
@@ -68,13 +68,13 @@ const traceData = {
     },
     {
```

Comment:
Note these are raw objects, not NetworkRequest instances

Reply:
we should really have converted these by now :/

Reply:
done!